docs/technical_reference/unit_models/uv_aop.rst

```diff
@@ -95,10 +107,19 @@ Equations and Relationships
    "Pseudo-first order rate constant", ":math:`k_0 = I \cdot k`"
    "Pseudo-first order rate constant", ":math:`k_0 = k_d + k_i`"
    "Solvent mass balance", ":math:`M_{\text{H2O},out} = M_{\text{H2O},in}`"
-   "Solute mass balance", ":math:`M_{j,out} = M_{j,in} \cdot \exp(D \cdot k)`"
-   "Electricity demand", ":math:`E_j = EE/O_j \cdot F_{in} \cdot \log_{10}(M_{j,in} / M_{j,out}) / \eta`"
+   "Solute mass balance", ":math:`M_{p, j, out} = M_{p, j, in} \cdot \exp(D \cdot k)`"
+   "Electricity demand of each component with phase", ":math:`E_{p, j} = EE/O_{p, j} \cdot F_{in} \cdot \log_{10}(M_{p, j, in} / M_{p, j, out}) / \eta`"
+   "Electricity demand of each component", ":math:`E_j = \max_p E_{p, j}`"
+   "Electricity demand", ":math:`E = \max_j E_j`"
    "UV absorbance", ":math:`UVA = -\log_{10}(UVT)`"
 
+if ``uv_dose_type`` is set to ``UVDoseType.calculated``:
+
+.. csv-table::
+   :header: "Description", "Equation"
+
+   "UV dose", ":math:`D = 10^A \cdot (-\log_{10}(UVT))^(-B \cdot \log_{10}(UVT)) \cdot (\frac{S}{S_0} / F_{in})^C \cdot N_{bank}^D`"
```

Comment:
The exponent is messed up for B*log10(UVT). I think you can try wrapping in curly braces:
```suggestion
   "UV dose", ":math:`D = 10^A \cdot (-\log_{10}(UVT))^{(-B \cdot \log_{10}(UVT))} \cdot (\frac{S}{S_0} / F_{in})^C \cdot N_{bank}^D`"
```